### PR TITLE
Cycle4/shape/upgrade alchemy

### PR DESCRIPTION
**Pull request by pascalbetz**
Mainly a merge of upstream 6.0 stable to our last branch.
And then some fixes (check my last commits).

Do not merge (since it would force new deploys of our app to use this version already), this is why i marked this as a draft.

### CODE REVIEW

File: app/controllers/alchemy/admin/essence_videos_controller.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Alchemy
+  module Admin
+    class EssenceVideosController < Alchemy::Admin::BaseController
```
Comment (ZmagoD) [inline app/controllers/alchemy/admin/essence_videos_controller.rb:5]: now that we had a video essence we can remove the custom one that @dancristianb added 🤔

**Review by ZmagoD — APPROVED** [review]
It is hard to understand what is from the merge and what custom code. I will do a functionality review in the other PR

File: app/controllers/alchemy/admin/essence_videos_controller.rb (hunk 1 above)
Comment (dancristianb) [reply app/controllers/alchemy/admin/essence_videos_controller.rb:5]: Yep, I vote we track this under a cooldown card so we don't overload this PR 👍

**Review by dancristianb — APPROVED** [review]
Looks good! 🚀

**Pull request closed without merging**